src/routes/event-team.tsx

```diff
@@ -156,12 +157,22 @@ const teamHeadingSpanStyle = css`
   margin-right: 0.3rem;
 `
 
+const headerLinkStyle = css`
+  color: white;
+
+  &:hover {
+    color: ${greenOnPurple};
```

Comment:
@varCepheid I have a hard time with the green text on purple background. To me, it doesn't seem to match very well. Also, there is not very good contrast between the hover text color and the background color, which is an accessibility concern. https://developer.mozilla.org/en-US/docs/Web/Accessibility/Understanding_WCAG/Perceivable/Color_contrast

I tried out some other colors to see what might look a little better and be more legible in terms of contrast. Light blue seems a little better to me, but it still didn't look quite right to me. Ultimately I think that white text on purple just looks a lot better. I don't think that the hover style in this case is really necessary, so I'd be happy with just letting the text remain white on hover. If having some sort of hover effect is important to you, maybe can we try setting `text-decoration-color` instead? It would be a little more subtle and wouldn't hurt contrast.